.github/workflows/workflow-dispatch-standalone-group-linux.yml

```
@@ -0,0 +1,43 @@
+name: "Workflow/Dispatch/StandaloneGroup/Linux"
+
+defaults:
+  run:
+    shell: bash --noprofile --norc -euo pipefail {0}
+
+on:
+  workflow_call:
+    inputs:
+      job-array:
+        description: "The dispatch.json's linux_standalone.jobs.<name> array of dispatch jobs."
+        type: string
+        required: true
+
+jobs:
+  run-jobs:
+    name: "${{ matrix.name }}"
+    strategy:
+      fail-fast: false
+      matrix:
+        include: ${{ fromJSON(inputs.job-array) }}
+    permissions:
+      id-token: write
+      contents: read
+    runs-on: ${{ matrix.runner }}
+    container:
+      image: cruizba/ubuntu-dind:jammy-26.1.3 # See workflow-run-job-linux for details
+      env:
+        NVIDIA_VISIBLE_DEVICES: ${{env.NVIDIA_VISIBLE_DEVICES}}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+      - name: Run job
+        uses: ./.github/actions/workflow-run-job-linux
```

Should we check out to the `cccl` path here, and then remove the `actions/checkout` step from the `workflow-run-job-linux` action?

```suggestion
          persist-credentials: false
          path: ${{github.event.repository.name}}
      - name: Run job
        uses: ./${{github.event.repository.name}}/.github/actions/workflow-run-job-linux
```

---

This checkout [uses the GitHub REST API](https://github.com/NVIDIA/cccl/actions/runs/9421977570/job/25957408428?pr=1797#step:4:23) (because `git` is not in the container), and the one later [uses `git fetch`](https://github.com/NVIDIA/cccl/actions/runs/9421977570/job/25957408428?pr=1797#step:5:190).

If the checkout step that uses the REST API doesn't include the `.git` directory, then that's a reason to do the two checkout steps.

---

Agreed, this was something I wanted to clean up. I'll give this a go.

---

This doesn't work -- we can't use `${{}}` in `uses:`, and I don't want to hardcode 'cccl' since I want to reuse this from NVBench. I'll look into fixing this in a followup.